src/components/AddBook.js

```diff
@@ -26,7 +26,8 @@ const AddBook = () => {
 
   return (
     <>
-      <h5>ADD NEW BOOK</h5>
+      <hr className="line" />
+      <h5 style={{ color: 'gray' }}>ADD NEW BOOK</h5>
```

Comment:
- [ ] It would be better if we follow the CSS best practices. React inline styling is allowed but it would be better if we create a variable and pass it to the element. For example; `const colGray = { color: 'gray'}`, `<h5 style={colGray}>`. Kindly do this for the other files